optaweb-tsp-ui/package-lock.json

```diff
@@ -10770,6 +11102,12 @@
         }
       }
     },
+    "underscore": {
+      "version": "1.4.4",
+      "resolved": "https://registry.npmjs.org/underscore/-/underscore-1.4.4.tgz",
+      "integrity": "sha1-YaajIBBiKvoHljvzJSA88SI51gQ=",
+      "dev": true
+    },
```

Comment:
@danielefiungo Can you explain these changes in `package-lock.json`? If I run `mvn clean install` locally, they get remove.

Reply:
Probably is related to test implementation I've add:
"enzyme": "^3.6.0" and "enzyme-adapter-react-16" without commit package.json to keep it separate. 
I'll revert it.

Reply:
Yeah, makes sense. Thank you!